_posts/2024-04-01-pyos-newsletter-april-2024.md

```diff
@@ -0,0 +1,70 @@
+
+Learn more about pyOpenSci’s open peer review process by reading through the [EOmaps submission](https://github.com/pyOpenSci/software-submission/issues/138) or checking out our [peer review page](https://www.pyopensci.org/about-peer-review/).
+
+## <i class="fa-solid fa-rocket"></i> Launched: pyOpenSci package showcase
+
+pyOpenSci currently has [28 packages](https://www.pyopensci.org/python-packages.html) that have been accepted through our open peer review process, and we want you to get to know them better! This is why we’ve launched our pyOpenSci package showcase, which highlights an accepted package and its author(s) every Tuesday and Thursday on our [Fosstodon](https://fosstodon.org/@pyOpenSci) and [LinkedIn](https://www.linkedin.com/company/pyopensci) pages.
+
+Have a package you’ve developed, but aren’t sure it’s in scope for pyOpenSci? Complete a [pre-submission inquiry](https://github.com/pyOpenSci/software-submission/issues/new?assignees=&labels=presubmission&projects=&template=presubmission-inquiry.md&title=) and one of our editors will be in touch!
+
+## <i class="fa-brands fa-python"></i> Upcoming Python Events for Scientists
+### PyConDE & PyData Berlin
+Online tickets are still available for [PyConDE & PyData Berlin](https://2024.pycon.de/), so be sure to grab them while they still last! The event is scheduled for April 22nd-24th, **with [Inessa Pawson](https://github.com/InessaPawson) delivering the keynote, titled “The Art and Science of Tending Open Source Orchards”**. You definitely don’t want to miss out!
+
+### PyCon US 2024
+[Registration is still open](https://us.pycon.org/2024/) for [PyCon US 2024](https://us.pycon.org/2024/), happening May 15th–23rd in Pittsburgh, PA. There’s also an online component for folks who can’t be there in person. It’s shaping up to be a stacked event, and you can view the schedule [here](https://us.pycon.org/2024/schedule/).
+
+### GeoPython
+Registration is live for [GeoPython](https://2024.geopython.net/), THE event for Python- and Geo-Enthusiasts. There are online and in-person options, and the event is scheduled for May 27th–29th in Basel, Switzerland.
+
+### SciPy–early bird ends soon!
+Happening July 8th–14th in Tacoma, WA, [SciPy registration is live](https://www.scipy2024.scipy.org/), and you can grab your early bird tickets until April 30th.
+
+Have an upcoming scientific Python event that’s open to the public? Email us at [media@pyopensci.org](media@pyopensci.org) with the details to have your event listed.
+
```

Comment:
```suggestion
Have an upcoming scientific Python event that’s open to the public? Email us at [media@pyopensci.org](mailto:media@pyopensci.org) with the details to have your event listed.

```

@kierisi when you create an email "link" you have to use mailto: in order for it to register as an email. otherwise it will just look like a link (that is broken) in the output file. CI flagged this as well. i'll go ahead and fix it and look at the build.